### ISSUE TITLE
Is there a way to disable warnings about units?

### ISSUE THREAD

**Issue by pie6k** [styled-components/css-to-react-native#137]
I know they should be there, I know the reason behind it, but I've got pretty large app and I don't want/can't codemod it right now. As I've got 100s of such components, it's making debugging pretty hard.

It would be great to have some flag to disable this warning.

![image](https://user-images.githubusercontent.com/7311462/76704107-08364e80-66d7-11ea-9cd4-8be147e2fcf2.png)

Another solution could be grouping them and showing something like `You've got 242 components without units`

As this warning does not include stack trace for the single component - it's not really any more helpful than having one main warning (as I have to look for those manually anyway which is also quite annoying)

If it'd be `Expected style "marginRight: 10" to contain units (src/ui/component.tsx)` I could at least command-click it to solve it one by one ad-hoc

**Comment by pie6k** [comment]
In case someone will need it, I've created simple file that will ignore some warning both in Yellowbox and terminal

```ts
import { YellowBox } from 'react-native';

const ignoredWarnings: string[] = ['Expected style'];

function isWarnIgnored(...args: any[]) {
  return args.some((arg) => {
    if (typeof arg !== 'string') {
      return false;
    }

    return ignoredWarnings.some((ignoredWarning) => {
      return arg.includes(ignoredWarning);
    });
  });
}

(function() {
  var _warn = console.warn;

  console.warn = function(...args: any[]) {
    if (isWarnIgnored(...args)) {
      return;
    }
    _warn(...args);
    // _warn.apply(console, ...args);
  };
})();

YellowBox.ignoreWarnings(ignoredWarnings);
```

**Comment by jacobp100** [comment]
The grouping is an interesting idea, but it would be quite hard to actually implement

Closing as your solution is the best workaround

**Comment by OKNoah** [comment]
As of `react-native@0.63.0` you must simply change two lines of @pie6k's code:

```
import {LogBox} from 'react-native'
LogBox.ignoreLogs(ignoredWarnings)
```
